src/odemis/driver/simulated.py

```diff
@@ -73,6 +73,7 @@ class Stage(model.Actuator):
     def __init__(self, name, role, axes, ranges=None, **kwargs):
         """
         axes (set of string): names of the axes
+        ranges (dict string -> float,float): min/max of the axis
```

Comment:
also add other params

Reply:
`name` and `role` are just the standard arguments. `kwargs` is just there to pass "hidden" arguments to the base Component. So there are no other arguments to describe.